cc.config.json

```diff
@@ -165,8 +155,7 @@
         "spine": {
             "modules": ["spine"],
             "dependentAssets": [
-                "c27215d8-6835-4b68-bfbb-bdeac6100c04",
-                "b5d6115f-0370-4d7c-aad3-c194cc71cf98"
```

Comment:
Can not delete it, Still has references.

![image](https://github.com/cocos/cocos-engine/assets/15109777/d4849d6d-e2f2-4499-a9f3-bab6f2359bb2)

Reply:
Spine has "b5d6115f-0370-4d7c-aad3-c194cc71cf98"，Dragon-bones also have "b5d6115f-0370-4d7c-aad3-c194cc71cf98"，Only one can be retained